.github/workflows/_meta.yaml

```
@@ -101,7 +101,10 @@ jobs:
         uses: shogo82148/actions-upload-release-asset@v1.5.0
         with:
           upload_url: ${{ github.event.release.upload_url }}
-          asset_path: ./artifact/*
+          asset_path: |
+            ./artifact/*.zip'
+            ./artifact/*.deb'
+            ./artifact/*.sha256sum'
```

Copypasta?
```suggestion
            ./artifact/*.zip
            ./artifact/*.deb
            ./artifact/*.sha256sum
```

Depending on how the glob works here, it may not pick up files in sub-directories.
I'm not sure if that ever matters here.

---

Bad copypasta and originally it was intentional to only grab on one level but I guess it doesn't hurt picking up recursivly.

Fixes didn't work anyway so I'll change it in the next try...